scripts/robust.yaml

```diff
@@ -0,0 +1,62 @@
+id: str(min=7, max=7)
+title: str(min=1, max=160)
+description: str(min=1)
+classification: str(min=1)
+keywords: list(str(min=1, max=80))
+system: str(min=1, max=40)
+severity: enum("error", "warning", "convention-violation", "bad-smell",
+            "minor-issue", "not-a-bug")
+links: list(str())
+bug: include('_bug')
+fix: include('_fix')
+time-machine: include('_time-machine')
+bugzoo: include('_bugzoo')
+
+---
+
+_bug:
+    phase: enum("programming-time", "build-time", "compile-time",
+            "deployment-time", "runtime-initialization", "runtime-operation")
+    specificity: enum("general issue", "robotics-specific", "ROS-specific",
+            "application-specific")
+    architectural-location: enum("application-specific code", "platform code", "N/A")
+    application: any(str(min=1), null())
+    task: enum("perception", "localization", "planning", "manipulation",
+            "human-robot interaction", "simulation", "diagnostics", "SLAM",
+            "N/A")
+    subsystem: str(min=1)
+    package: list(regex("^([a-z0-9A-Z-]+)(/[a-z0-9A-Z_]+){2}$"))
+    languages: list(enum("python", "cmake", "C++", "package.xml",
+                      "launch.xml", "msg", "srv", "xacro", "urdf", "robot specific"))
+    detected-by: enum("build system", "compiler", "code scanning tool",
+            "assertions", "runtime detection", "runtime crash",
+            "testing violation", "developer", "user")
+    reported-by: enum("guest user", "contributor", "member developer",
+                  "automatic", "unreported")
+    issue: any(regex("^((https://github\\.com/[a-zA-Z0-9_]+/[a-zA-Z0-9_]+/issues/[0-9]+)?)$"), str("N/A"))
+    time-reported: any(date(), null())
+    reproducibility: str()
+    trace: str(min=1)
+    reproduction: any(str(min=1), null(), required=False)
+
+_fix:
+    repo: regex("^(https://github\\.com/[a-zA-Z0-9-]+/[a-zA-Z0-9_]+)$")
+    hash: regex("^[0-9a-f]{40}$")
+    pull-request: regex("(^(https://github\\.com/[a-zA-Z0-9-]+/[a-zA-Z0-9_]+/pull/[0-9]+)?$)|(^N/A$)")
+    license: list(enum("BSD", "GPLv3", "LGPLv3"))
+    fix-in: list(str(min=1))
+    languages: list(enum("python", "cmake", "C++", "package.xml", "launch.xml",
+                  "msg", "srv", "xacro", "urdf", "robot specific"))
+    time: any(date(), null())
+
+_time-machine:
+    ros_distro: enum("noetic", "melodic", "lunar", "kinetic", "jade", "indigo",
+                  "hydro", "groovy", "fuerte", "electric", "diamondback")
+    ros_pkgs: list(str(min=1))
+    datetime: date()
```

Comment:
Does this require another package?

```
(robust) chris@chris-MS-7B77:~/bugs/robust$ python scripts/yamale_validator.py geometry2/b206807/b206807.bug 
Traceback (most recent call last):
  File "/home/chris/.local/share/virtualenvs/robust-q7eP5Wk_/local/lib/python3.7/site-packages/yamale/syntax/parser.py", line 16, in parse
    validators)
  File "<ast>", line 1, in <module>
NameError: name 'date' is not defined
```

Reply:
It looks like `date` has been renamed to `day`:
https://github.com/23andMe/Yamale#day---daymindate-maxdate

Reply:
Oh, sorry. My bad. It looks like I was somehow using an older version of `yamale_validator.py`!